### PR TITLE
fix(docs-ui): paste link on link

### PR DESCRIPTION
**Pull request by weird94**
close https://github.com/dream-num/univer-pro/issues/2552
close https://github.com/dream-num/univer-pro/issues/2293
close https://github.com/dream-num/univer-pro/issues/1992
close https://github.com/dream-num/univer-pro/issues/1620

## Pull Request Checklist

- [x] Related tickets or issues have been linked in the PR description (or missing issue).
- [x] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [x] Unit tests have been added for the changes (if applicable).
- [x] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/docs-ui/src/commands/commands/clipboard.inner.command.ts [hunk 1]
```diff
@@ -89,9 +89,9 @@ export const InnerPasteCommand: ICommand<IInnerPasteCommandParams> = {
         const commandService = accessor.get(ICommandService);
         const docSelectionManagerService = accessor.get(DocSelectionManagerService);
         const univerInstanceService = accessor.get(IUniverInstanceService);
-        const selections = docSelectionManagerService.getCurrentTextRanges();
+        const _selections = docSelectionManagerService.getCurrentTextRanges();
```
Comment (Jocs) [inline packages/docs-ui/src/commands/commands/clipboard.inner.command.ts]: Any better name than _selections?

**Review by Jocs — APPROVED** [review]

File: packages/sheets-ui/src/commands/operations/set-zoom-ratio.operation.ts [hunk 2]
```diff
@@ -31,8 +32,11 @@ export const SetZoomRatioOperation: IOperation<ISetZoomRatioOperationParams> = {
     handler: (accessor, params: ISetZoomRatioOperationParams) => {
         const renderManagerService = accessor.get(IRenderManagerService);
```
Comment (wzhudev) [inline packages/sheets-ui/src/commands/operations/set-zoom-ratio.operation.ts:33]: 这个需要在源头阻止操作的生成，担心在 live share 场景下会有问题：本地虽然没有缩放但是 follower 缩放了

**Review by wzhudev — CHANGES_REQUESTED** [review]

File: packages/sheets-ui/src/commands/operations/set-zoom-ratio.operation.ts (hunk 2 above)
Comment (Jocs) [reply packages/sheets-ui/src/commands/operations/set-zoom-ratio.operation.ts:33]: 感觉编辑的时候很多操作都应该阻止，不仅仅是缩放

Comment (Jocs) [reply packages/sheets-ui/src/commands/operations/set-zoom-ratio.operation.ts:33]: 之前编辑器好多bug都和在编辑态时，用户做了其他一些操作导致的，比如切换 sheet，放大缩小等。编辑cell和这些操作不能并行，要么关闭编辑器要么阻止操作？

**Pull request merged**